runtime-config.md

```diff
@@ -34,6 +34,12 @@ Additional filesystems can be declared as "mounts", specified in the *mounts* ar
         "source": "/volumes/testing",
         "destination": "/data",
         "options": ["rbind","rw"]
+    },
+    {
+        "type": "ext3",
+        "source": "qcow2:/volumes/data.qcow2",
+        "destination": "/data",
```

Comment:
Good point, It's looks good to add an independent section for hypervisor.
Thanks!